releasenotes/notes/logs-tag-dirname-at-creation-9da7c8f17a9b45bf.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    logs agent - tailed files discovered through a configuration entry with
```

Nit: I don't know we have a formal way to highlight changes with the Logs product, but I think we should have one (makes it easier to read in the changelog). We could use a convention similar to what we do for APM, like this:

```suggestion
    Logs: tailed files discovered through a configuration entry with
```

WDYT?